libs/island-ui/core/src/index.ts

```diff
@@ -1,7 +1,7 @@
 // Components
 export * from './lib/Pagination/Pagination'
 export * from './lib/AsyncSearch/AsyncSearch'
-export * from './lib/Button/Button'
+export { Button as ButtonDeprecated } from './lib/Button/Button'
```

Comment:
Would like to see the `useDeprecatedComponent` hook inside `Button`.

Since this is a global rename, the name of `ButtonDeprecated` (which is very descriptive) might be missed by developers.

Reply:
good point! this is the first step in switching to the new button, there is still some internal wiring needed in island-ui, we will surely add this in the next reiterations